templates/details/actions.html

```diff
@@ -79,4 +79,8 @@ <h4>No actions have been added yet</h4>
 
 {% block details_scripts %}
   <script src="{{ versioned_static('js/dist/highlight-nav.js') }}" defer></script>
+  <script>
+      // Fix to see the content under the sticky nav when you navigate using #id
+      window.addEventListener("hashchange", () => { scrollBy(0, -45) })
```

Comment:
would be nice to add a bit more space so the title is not attached directly to the navigation
![image](https://user-images.githubusercontent.com/2707508/110618008-376a8900-818e-11eb-8036-2bde3fcbf29d.png)

Something like this:
![image](https://user-images.githubusercontent.com/2707508/110618071-46e9d200-818e-11eb-843c-cd2f26305c0f.png)